docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -23,6 +23,11 @@ The config file is specified in JSON format. It contains rules that define
 which users have access to which resources. The rules are read from top to bottom
 and the first matching rule is applied. If no rule matches, access is denied.
 
+.. note::
+
+    It enforces access control on Trino coordinator, these configuration files for access
```

Comment:
change to `Access control is enforced on Trino coordinator, therefore access control configuration files are only required on coordinator node`